installer/dbscript/dbscript-2.sql

```diff
@@ -3564,6 +3564,8 @@ UPDATE `ohrm_menu_item` SET `status` = '0' WHERE `ohrm_menu_item`.`menu_title` =
 UPDATE `ohrm_menu_item` SET `status` = '0' WHERE `ohrm_menu_item`.`menu_title` = 'Reports' AND `screen_id` IS NOT NULL;
 UPDATE `ohrm_menu_item` SET `status` = '0' WHERE `ohrm_menu_item`.`menu_title` = 'Data Import';
 
+UPDATE `ohrm_menu_item` SET `status` = '1' WHERE `ohrm_menu_item`.`menu_title` = 'Leave Types';
```

Comment:
remove this line. menu items enable once leave period defined. Hope data group permission already defined for this API.

Reply:
removed